ALevel HW7/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ALevel_HW7.Models;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        Person person = new Person("Maksym Duzhyi", 18);
+        person.DisplayInfo();
+
+        Console.WriteLine();
+
+        Teacher teacher = new Teacher("Petro Chernych", 25, "Teacher", ".Net/C#");
+        teacher.DisplayInfo();
+
+        Console.WriteLine();
+
+        NonTeachingStaff nonTeachingStaff = new NonTeachingStaff("John Weak", 32, "Cleaner", "Caretaker");
+        nonTeachingStaff.DisplayInfo();
+
+        Console.WriteLine();
+
+        Student srudent = new Student("Melisa Jenson", 24, 8);
+        srudent.DisplayInfo();
+
+        Console.WriteLine();
+
+        Course course = new Course(".Net/C#", 9);
+        course.DisplayInfo();
```

Comment:
Nice, but it would be much more interesting, if I were able to choose whether I'd like to see info about Teacher, Student and so on.